SmartPlaces.Facilities/lib/IngestionManager.Mapped/README.md

```diff
@@ -1,12 +1,53 @@
 # IngestionManager.Mapped
 
-This library provides a way to load a DTDL based graph from a Mapped Graph
+This library works in conjunction with and depends on the `Microsoft.SmartPlaces.Facilities.IngestionManager` library, that defines interfaces and a generic implementation for ingesting building data graphs into Azure Digital Twins. 
 
+## Setup
 
-## Interfaces and Classes
+This library provides `IInputGraphManager` and `IGraphIngestionProcessor` implementations and a subclass of `IngestionManagerOptions` that together allow solutions to ingest graphs from the [Mapped](https://app.mapped.com/) API. 
 
-### Class: MappedGraphIngestionProcessor
+Those implementations are wired up using the .Net dependency injection framework, as per the common design pattern:
 
-*Description*
+```csharp
+services.AddMappedIngestionManager(options =>
+    {
+        // Mapped Specific
+        options.MappedToken = hostContext.Configuration["MappedToken"];
+        options.MappedRootUrl = hostContext.Configuration["MappedRootUrl"];
 
-Implementation of IGraphIngestionProcessor class for loading a site graph from Mapped API. Implements IGraphIngestionProcessor
+        // Ingestion Manager
+        options.AzureDigitalTwinsEndpoint = hostContext.Configuration["AzureDigitalTwinsEndpoint"];
+    });
+```
+
+The above sets up singleton implementations such that any code depending on `IInputGraphManager` will call `MappedGraphManager`, and code depending on `IGraphIngestionProcessor` will call `MappedGraphIngestionProcessor`. 
+
+Note that for ontology mappings to be resolved correctly, you may also want to wire up an ontology mapper instance:
+
+```csharp
+services.AddLogging();
+
+services.AddSingleton<IOntologyMappingLoader>(sp =>
+{
+    var logger = sp.GetRequiredService<ILogger<MappedOntologyMappingLoader>>();
+    return new MappedOntologyMappingLoader(logger, hostContext.Configuration["ontologyMappingFilename"]);
+});
+
+services.AddSingleton<IOntologyMappingManager, OntologyMappingManager>();
+```
+
+## Usage
+
+The entry point to using these libraries is the base class `IngestionProcessorBase.IngestFromApiAsync()` method; that method in turn calls the `MappedGraphIngestionProcessor.GetSites()` method, which initiates ingestion of all sites (e.g., campuses/buildings or other suitable starting nodes) from the input graph.
```

Comment:
Just calling out that this GetSites could fall out of sync with your other PR

Reply:
Good call-out, once the other refactoring PRs are merged I'll update this one accordingly.